app/models/provider_tag_mapping.rb

```diff
@@ -22,7 +22,7 @@ class ProviderTagMapping < ApplicationRecord
 
   require_nested :Mapper
 
-  TAG_PREFIXES = %w(amazon azure kubernetes openstack).map { |name| "/managed/#{name}:" }.freeze
+  TAG_PREFIXES = %w(amazon azure kubernetes ibm openstack vmware).map { |name| "/managed/#{name}:" }.freeze
```

Comment:
would be nice if this were pluggable by getting the value from each provider that supports tag mapping

Reply:
Yes I've already added a checklist item for this to the pluggability issue